products/compute/api.yaml

```
@@ -239,6 +272,66 @@ objects:
           last character, which cannot be a dash.
         input: true
         required: true
+  - !ruby/object:Api::Resource
```

```suggestion
  // This is a fine-grained resource
  - !ruby/object:Api::Resource
```

Since this is the first fine-grained resource, let's add a comment so readers understand what we're seeing until we work out some patterns for them